lib/order.rb

```diff
@@ -0,0 +1,63 @@
+require './lib/menu.rb'
+require './lib/verify.rb'
+
+class Order
+
+attr_reader :current_order, :current_menu, :no_of_item, :item_total, \
+            :collect_amount, :text_message
+
+    def initialize(menu_klass: Menu, verify_klass: Verify)
+      @current_order = []
+      @current_menu = []
+      @menu_klass = menu_klass
+      @verify_klass = verify_klass
+
+    end
+
+    def start_order(item, no_of_item)
+     current_menu << menu_klass.new
+     current_order << current_menu[0].menu.select {|k,v| k == item}
+     current_order[-1][:no_of_item] = no_of_item
+     current_order[-1][:item_total] = (current_order[-1][item] \
+     * current_order[-1][:no_of_item])
+
+    end
+
+    def next_item(item, no_of_item)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for next_item is too high. [20/15] (http://c2.com/cgi/wiki?AbcMetric)